README.md

```diff
@@ -4,31 +4,30 @@ QGIS plugin for drone image classification using machine learning.
 
 ## Installation (prototype)
 
-Dowload the [ZIP file of DroneML](https://nlesc-my.sharepoint.com/:u:/g/personal/o_ku_esciencecenter_nl/ETVp8wcBhJ1Com3PDohe5eQB219nCHs6wJURJymwqR6uRw?e=lOYkvn).
+### Prepare Python environment in QGIS (pre-requisite)
 
 In QGIS, go to `Plugins` > `Python Console`.
 
 Run the following Python code to install the dependencies required by the plugin:
 
 ```python
-import pip
-pip.main(["install", "tqdm", "rioxarray", "geopandas", "scikit-learn", "torch", "torchvision", "torchinfo"]
+!pip install -U segmentmytif
 ```
+
 Restart QGIS for changes to your Python package installation to take effect.
-Then, go to `Plugins` > `Manage and Install Plugins...` > `Install from ZIP` and select the downloaded `droneml.zip` file   
 
-### Manual Installation if install from ZIP fails
+### Install DroneML plugin from ZIP
 
-Manually unzip the `droneml.zip` file to folder `droneml`. And `cd` into the `droneml` folder.
+Dowload the [ZIP file of DroneML](LINK_TO_GITHUB_RELEASE/TO_BE_ADDED).
```

Comment:
There is another option. We can download the the zip of the current repo using https://github.com/DroneML/DroneML/archive/refs/heads/main.zip

Reply:
Wow! I did not know about this. Quite useful! But in the doc, I would still like to put in the release zip, since it gives control on the version. What do you think?